pwa/pages/about/index.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import Link from 'next/link';
+import { NextPage } from 'next';
+import { List, Subtitle, Title } from "components/common/text";
+
+const About: NextPage = () => {
+  return (
+    <div className="max-w-4xl mx-auto px-4 py-8">
```

Comment:
Why `max-w-4xl`?
Why `px` and `py`? The page padding should be handled by the layout.